metaci/testresults/tests/test_robot_importer.py

```diff
@@ -514,3 +514,26 @@ def test_gus_bus_payload_metadata():
             "key1": True,
             "key2": "hello, world",
         }
+
+
+@responses.activate
+@pytest.mark.django_db
+def test_gus_bus_bad_payload_metadata():
+    """Verify that all build-specific metadata is added to the payload"""
+    flowtask = FlowTaskFactory()
+    flowtask.build_flow.build.org = OrgFactory()
```

Comment:
You shouldn't need this line; we changed the BuildFactory so that it automatically gets created with an org: https://github.com/SFDO-Tooling/MetaCI/blob/9dbddc76bc85e68d08568e9695828f126bf5264c/metaci/fixtures/factories.py#L115